### PR TITLE
AC-791 Updated UI of registeration screen

### PR DESCRIPTION
**Pull request by rishabh-997**
## Description of what I changed
1. Converted maximum XML to constraint layout
2. Changed UI
3. Renamed id's to so they make more sense
4. Used view binding in whole activity
5. Updated Alert dialog box

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-791

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
PicsArt_06-23-10 34 10

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/logs/LogsFragment.kt [hunk 1]
```diff
@@ -50,7 +50,7 @@ class LogsFragment : ACBaseFragment<LogsContract.Presenter>(), LogsContract.View
     private fun setClipboard(context: Context?, text: String?) {
         val clipboard = context!!.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
         val clip = ClipData.newPlainText(getString(R.string.copied_text), text)
-        clipboard.primaryClip = clip
+        clipboard.setPrimaryClip(clip)
     }
```
Comment (LuGO0) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/logs/LogsFragment.kt:54]: @rishabh-997 this change got merged right in the previous PR?? did you forget  to rebase :) ??

Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/logs/LogsFragment.kt:54]: Oh, didn't notice, will revert it

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java [hunk 2]
```diff
@@ -118,44 +109,15 @@
 @RuntimePermissions
 public class AddEditPatientFragment extends ACBaseFragment<AddEditPatientContract.Presenter> implements AddEditPatientContract.View, CameraOrGalleryPickerDialog.onInputSelected {
 
-    private RelativeLayout relativeLayout;
-    private LocalDate birthdate;
+    private final static int IMAGE_REQUEST = 1;
+    private final static int GALLERY_IMAGE_REQUEST = 2;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java:113]: Remove these as they're in ApplicationConstants now.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java [hunk 3]
```diff
@@ -804,12 +719,14 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
...
+            if (resultCode == Activity.RESULT_OK) {
+                Uri sourceUri = data.getData();
+                File dir = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DCIM);
+                output = new File(dir, getUniqueImageFileName());
+                Uri destinationUri = Uri.fromFile(output);
+                openCropActivity(sourceUri, destinationUri);
+            }
         } else if (requestCode == UCrop.REQUEST_CROP) {
             if (resultCode == Activity.RESULT_OK) {
                 patientPhoto = getResizedPortraitImage(output.getPath());
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java:732]: Change line 733-735 to

```java
Bitmap bitmap = ThumbnailUtils.extractThumbnail(patientPhoto, binding.patientPhoto.getWidth(), binding.patientPhoto.getHeight());
binding.patientPhoto.setImageBitmap(bitmap);
binding.patientPhoto.invalidate();
```

**Pull request merged**